contracts/gas-service/AxelarGasService.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.8.0;
 
 import { IERC20 } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IERC20.sol';
```

Comment:
need to bump gmp sdk release

Reply:
In package.json? Not released yet, right?